PlayCover/Views/AppSettingsView.swift

```diff
@@ -523,12 +560,16 @@ struct MiscView: View {
                     }
                     .frame(width: 225)
                     .onChange(of: applicationCategoryType) { _ in
+                        task = .applicationCategoryType
                         app.info.applicationCategoryType = applicationCategoryType
                         Task(priority: .userInitiated) {
-                            do {
-                                try Shell.signApp(app.executable)
-                            } catch {
-                                Log.shared.error(error)
+                            DispatchQueue.global(qos: .background).async {
```

Comment:
This `DispatchQueue` inside the `Task` isn't necessary.

Reply:
According to my observation, when only `Task` is used, UI will respond to all operations performed by the user during the waiting period after the end of the blocking, which may cause some functions be accidentally triggered, so I chose to use `DispatchQueue.global(qos: .background)`. In the latest commit, I removed the `Task` in this code to ensure simplicity.

Reply:
What UI operations? I can't seem to replicate this.

Reply:
I recorded two videos to compare and show the differences in the results of these two codes with only the following differences (I wrote a demo program to show the problems more clearly, and they behave exactly the same in PlayCover)
`Task(priority: .userInitiated) { doSomething() }` 
https://github.com/PlayCover/PlayCover/assets/30586070/9c579568-9a92-4e89-b806-575366b69b13

`DispatchQueue.global(qos: .background).async { doSomething() }`
https://github.com/PlayCover/PlayCover/assets/30586070/c21ed8b5-25c8-4116-9a61-c5179ccf80c2

Reply:
Try `Task.detached` instead of `Task(priority: .userInitiated)`.

Reply:
Wow, that looks better. I have added this change to the latest commit.